tutorials/xr/xr_action_map.rst

```diff
@@ -385,3 +385,124 @@ This is why many XR runtimes only use it as a last resort and will attempt to us
   This is our advice as well: limit your action map to the interaction profiles for devices you have actually tested your game with.
   The Oculus Touch controller is widely used as a fallback controller by many runtimes.
   If you are able to test your game using a Meta Rift or Quest and add this profile there is a high probability your game will work with other headsets.
+
+Binding Modifiers
+-----------------
+
+While one of the main goals of the action map is to remove the need for the application to know the hardware used, sometimes the hardware has physical differences that require inputs to be altered in ways other than how they are bound to actions. This need ranges from setting thresholds, to altering the inputs available on a controller.
+
+Currently OpenXR supports two binding modifiers that have been implemented in Godot but more are likely to follow in the future.
+
+These binding modifiers are not enabled by default and require enabling in the OpenXR project settings. Also there is no guarantee that these modifiers are supported by every runtime. You will need to consult the support for the runtimes you are targeting and decide whether to rely on the modifiers or implement some form of fallback mechanism.
```

Comment:
```suggestion
These binding modifiers are not enabled by default and require enabling in the
OpenXR project settings. Also there is no guarantee that these modifiers are
supported by every runtime. You will need to consult the support for the
runtimes you are targeting and decide whether to rely on the modifiers or
implement some form of fallback mechanism.
```
One other thing, which applies to this whole PR.

In the manual the rule is usually to wrap long lines by hand after 80-100 characters. This was an informal rule for a long time and I recently [wrote it down](https://docs.godotengine.org/en/latest/contributing/documentation/docs_writing_guidelines.html#manually-wrapping-lines). (I'd say wrapping at 80 is ideal and wrapping after 100 is mandatory. Allowing some room between 80 and 100 reduces some diff noise since later minor changes don't *force* a re-wrap.)

As it stands, the manual isn't exactly standardized. But I think we should avoid introducing *new* very long lines like this one.

Reply:
(I also think that it would be very nice if we could avoid this rule somehow, and instead use one line per paragraph and let the tools automatically wrap, like the class reference XML. But I don't know if the tooling support is quite there yet.

Reply:
Yeah its annoying to follow through on this, especially when you're still editing things and suddenly the linebreaks need to be in different places. Not to mention the trailing spaces that you easily introduce.

But I'll try and make sure the document is properly hard-wrapped.